pyproject.toml

```diff
@@ -25,7 +25,7 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = "^3.6"
-"ruamel.yaml" = "^0.16"
+"ruamel.yaml" = ">= 0.16, <0.18"
```

Comment:
Why `<0.18`?

Reply:
I did not want to be over permissive

Reply:
Let me elaborate, I did not want to be so permissive not knowing your initial intention